puppeteer/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "example-vue-puppeteer-app",
+  "version": "1.0.0",
+  "description": "Mocha and Puppeteer example",
+  "scripts": {
+    "start": "serve .",
+    "test": "mocha test/*.test.js"
```

Comment:
Please rename file.to `spec.js` and then can just run `mocha`. I think it can confuse people why it's needed to have this.

Reply:
`"test": "mocha test/*.spec.js"` like this? and `index.test.js` file rename  `index.spec.js` ?

Reply:
As in `"test" : "mocha"`. Its better practice to omit options on CLI unless they are really needed.